src/Viewer.tsx

```diff
@@ -149,7 +149,7 @@ function Viewer(): ReactElement {
   };
   const [notificationApi, notificationContextHolder] = notification.useNotification(notificationConfig);
 
-  const { bannerElement, showAlert } = useAlertBanner([dataset]);
+  const { bannerElement, showAlert, clearBanners } = useAlertBanner();
```

Comment:
Manually clear the banners instead of clearing whenever `dataset` changes. This caused a bug because the `useEffect` that listens for changes to `dataset` would fire immediately after the first render, causing the banners emitted by any error handlers to be cleared before they could be shown to the user.